docs/business-finder-links-based-facets.md

```diff
@@ -0,0 +1,40 @@
+# EU Exit Business Readiness finder links-based facets
+
+This document explains how the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder and retrieve content from different publishing apps.
```

Comment:
As mentioned in another comment (which seems to have disappeared), it's not accurate to say "content-tagger links". Yes, they're tagged in content-tagger, but finder-frontend doesn't care about that, only that the facets are in the content-item "links".